lib/passport-configurator.js

```diff
@@ -513,7 +513,8 @@ PassportConfigurator.prototype.configureProvider = function(name, options) {
       var func = (link ? passport.authorize : passport.authenticate).bind(passport);
       func(name, _.defaults({
         state: encodeURIComponent(req._parsedOriginalUrl.search || ''),
```

Comment:
Wait, we should remove the `callbackURL` and `scope` from this object, plus, correct the `callbackURL` to be `callback_url` in query string.